scripts/get_market_pairs.py

```diff
@@ -1,6 +1,8 @@
 import os
 import sys
 
+from freqtrade.exchange import is_exchange_supported, supported_exchanges
```

Comment:
I'm not certain on changing this script...
While the improvement might make sense - it's really a 1:1 copy of 
 one of the examples in ccxt [here](https://github.com/ccxt/ccxt/blob/master/examples/py/arbitrage-pairs.py) - which (i notice now) we should mention at the top...

Reply:
Did not know of that example in ccxt repo. Well, the target was to incapsulate (limit) importing of ccxt to the exchange module only.

Reply:
What do you think -- should we remove changes made in this script (since it's simply borrowed from external sources) and apply the rest?

Reply:
Yes, sounds like a plan (sorry, forgot about this ...)